src/context/style/layout.css

```diff
@@ -183,978 +154,4 @@ hr
 }
 
 
-/* a bunch of common div and span settings */
-
-div.sm_cell
-{
-    width: 4em;
-    float: left;
-}   
-
-div.med_cell
-{
-    width: 14em;
-    float: left;
-} 
-
-div.lg_cell
-{
-    width: 24em;
-    float: left;
-}  
-
-div.right
-{
-    text-align: right;
-}
-
-div.centered 
-{
-    text-align: center;
-}
-
-div.header {
-    
-    font-size: 1.5em;
-    font-weight: bold;
-    font-style: italic;
-    color: #555;
-    margin-bottom: 14px;
-    padding-left: 10px;
-}
-
-
-/* overlay styles */
-div.overlay_top {
-    position: absolute;
-    z-index: 4;
-    top: 140px;
-    left: 10%;
-    width: 70%;
-    padding: 10px;
-    border-width: 1px;
-    border-style: solid;
-    border-color: #000000;
-    background: #f0f0f0;
-    display: none;
-}
-
-
-div.overlay_dimmer {
-    position: fixed;
-    z-index: 3;
-    top: 0px;
-    left: 0px;
-    width: 100%;
-    height: 100%;
-    background: #999999;
-    filter: alpha(opacity=60); /* for IE compatibility, must occur before opacity */
-    opacity: 0.6;
-    display: none;
-}
-
-div.monitor {
-    padding: 2px;
-    margin: 4px 0 4px 0;
-}
-
-span.small
```

Comment:
these span.small med large classes are used in diff places in the tactic source code.. What's the workaround if they are removed?